Changed from Request::instance() to Request::current()

gjcourtney opened the pull request:
Updated the static calls to the Request object so that they now use the Kohana 3.3 APIs.

Commit:
Changed from Request::instance() to Request::current()

Updated the static calls to the Request object so that they now use the Kohana 3.3 APIs.

sergeyklay commented on classes/captcha.php:
```diff
@@ -434,10 +434,10 @@ public function image_render($html)
+        Request::current()->headers['Content-Type'] = 'image/'.$this->image_type;
+        Request::current()->headers['Cache-Control'] = 'no-store, no-cache, must-revalidate, post-check=0, pre-check=0';
+        Request::current()->headers['Pragma'] = 'no-cache';
+        Request::current()->headers['Connection'] = 'close';
```
Maybe like this?

``` php
// Send the correct HTTP header
Request::current()->response()
    ->headers('Content-Type', 'image/'.$this->image_type)
    ->headers('Cache-Control', 'no-store, no-cache, must-revalidate, post-check=0, pre-check=0')
    ->headers('Pragma', 'no-cache')
    ->headers('Connection', 'close');
```

Commit:
Updated to headers() calls to use Kohana style

Now a single chained call rather than 4 separate calls.

Commit:
Fix for dodgy commit

Previous commit didn't actually work - Request::current()->headers() now used rather than Request::current()->response()->headers

Commit:
Updates for Kohana 3.3

Updated to use UTF8:: rather than utf8:: static calls.

Commit:
Rename captcha.php to Captcha.php

File now starts with uppercase letter for Kohana 3.3

Commit:
Rename alpha.php to Alpha.php

Commit:
Rename basic.php to Basic.php

Commit:
Rename black.php to Black.php

Commit:
Rename math.php to Math.php

Commit:
Rename riddle.php to Riddle.php

Commit:
Rename word.php to Word.php

Commit:
Controller call update for Kohana 3.3

First letter now uppercase for Captcha

sergeyklay commented:
:+1:
